sdk/src/main/java/com/dailymotion/android/player/sdk/PlayerWebView.java

```diff
@@ -70,7 +77,15 @@
     public static final String EVENT_END = "end";
     public static final String EVENT_CONTROLSCHANGE = "controlschange";
     public static final String EVENT_VOLUMECHANGE = "volumechange";
+
+    /**
+     * @deprecated Use EVENT_QUALITY_CHANGE instead
+     */
+    @Deprecated
     public static final String EVENT_QUALITY = "qualitychange";
+    public static final String EVENT_QUALITY_CHANGE = "qualitychange";
+    public static final String EVENT_QUALITIES_AVAILABLE = "qualitiesavailable";
```

Comment:
Shouldn't all the `EVENT_XYZ` be private ? Or at least deprecated so that backward compatibility is kept ? We would rather have users check the instance type of the event=, right ?

Reply:
As discussed with you: we will those public for now